### PR TITLE
README polishing

### PR DESCRIPTION
**Pull request by ExoticMarbles**
I fixed some broken links, provided a YT link to a video of the Demo Mode unlocking gesture, and made some small readability changes.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,85 +1,73 @@
...
-
-Your files **will** be deleted make sure to make a backup.
+The most up to date release can be found [here](https://github.com/notmarek/LanguageBreak/releases/latest).
 
 # Installation
 
-## Before jailbreak
+## Warnings
+- ⚠️ This method **will** __DELETE__ all content on your device. Backup important files.
+- ⚠️ This method works up to firmware version `5.16.2.1.1`. Downgrade if needed.
```
[the remark this answers is not in the document]
Comment (ExoticMarbles) [reply README.MD:17]: Gotcha. Sorry, I may have taken some liberties when re-writing. I just loved the method and I wanted it to be more accessible to newbies like me. Let me rephrase it...

**Comment by notmarek** [comment]
<3
